VagrantProvisionUbuntu1604.sh

```diff
@@ -202,67 +202,9 @@ if ! grep --quiet "PATH=" ~/.profile; then
     source ~/.profile
 fi
 
-if ! ruby -v | grep --quiet 2.3.0; then
-    # Ruby via rvm - from rvm.io
-    gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 2>&1
-
-    curl -sSL https://raw.githubusercontent.com/rvm/rvm/master/binscripts/rvm-installer | bash -s stable
-
-    source ~/.rvm/scripts/rvm
-
-    stdbuf -o L -e L rvm install ruby-2.3
-    rvm --default use 2.3
-
-# Don't install documentation for gems
-cat > ~/.gemrc <<EOT
-  install: --no-document
-  update: --no-document
-EOT
-fi
-
-# gem installs are *very* slow, hence all the checks in place here to facilitate debugging
-if ! gem list --local | grep -q mime-types; then
-    echo "Gem Install: mime-types"
-   #sudo gem install mime-types -v 2.6.2
-   gem install mime-types
-fi
-
-if ! gem list --local | grep -q cucumber; then
-    echo "Gem Install: cucumber"
-   #sudo gem install cucumber
-   gem install cucumber
-fi
-
-if ! gem list --local | grep -q capybara-webkit; then
-    echo "Gem Install: capybara-webkit"
-   sudo apt-get install -y qt5-default libqt5webkit5-dev gstreamer1.0-plugins-base gstreamer1.0-tools gstreamer1.0-x
-   #sudo gem install capybara-webkit
-   gem install capybara-webkit
-fi
-
-if ! gem list --local | grep -q selenium-webdriver; then
-    echo "Gem Install: selenium-webdriver"
-   #sudo gem install selenium-webdriver
-   gem install selenium-webdriver
-fi
-
-if ! gem list --local | grep -q rspec; then
-    echo "Gem Install: rspec"
-   #sudo gem install rspec
-   gem install rspec
-fi
-
-if ! gem list --local | grep -q capybara-screenshot; then
-    echo "Gem Install: capybara-screenshot"
-   #sudo gem install capybara-screenshot
-   gem install capybara-screenshot
-fi
-
-if ! gem list --local | grep -q selenium-cucumber; then
-    echo "Gem Install: selenium-cucumber"
-   #sudo gem install selenium-cucumber
-   gem install selenium-cucumber
-fi
+# Use RVM to install the desired Ruby version, then install the gems.
+$HOOT_HOME/scripts/ruby/rvm-install.sh
+$HOOT_HOME/scripts/ruby/gem-install.sh
```

Comment:
This begs the question of how we are handling the ubuntu16.04 build.  Do we test these changes?  Or do we rely on the team utilizing the ubuntu16.04 build to test these changes on their own?

Reply:
Currently, we rely on those that are using 1604.

Reply:
I was under the impression that the Ubuntu 16.04 script is incomplete (see #1881).  I'm providing the change as a courtesy, and while untested I think it'll work fine.

Reply:
:+1: